script.js

```diff
@@ -1,11 +1,25 @@
+function openNavigation() {
+ const links = document.getElementById("mylinks");
+ const bar = document.getElementById("bar")
+  if (links.style.display === "flex") {
+    links.style.display = "none";
+    bar.classList.remove("fa-times");
+    bar.classList.add("fa-bars");
 
-const navigation = document.getElementById("nav");
-const menu = document.getElementById("menu");
+  } else {
+    links.style.display = "flex";
+    links.style.flexDirection="column"
+    bar.classList.remove("fa-bars");
```

Comment:
- [ ] Good job by implementing some requested changes from the previous reviewer. you're almost there. But, I think it would be better if you make invisible the hamburger menu when the menu bar appears by clicking. 
![image](https://user-images.githubusercontent.com/71874425/167653478-6218923a-70ec-4c9f-8335-08289edbed0e.png)
Kindly, fix this issue.